packages/web/src/components/leaving-audius-modal/LeavingAudiusModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useCallback } from 'react'
+
+import { useLeavingAudiusModal } from '@audius/common'
+import {
+  HarmonyButton,
+  HarmonyButtonType,
+  IconExternalLink,
+  IconInfo,
+  Modal,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalTitle
+} from '@audius/stems'
+
+import { HelpCallout } from '../help-callout/HelpCallout'
+
+import styles from './LeavingAudiusModal.module.css'
+
+const messages = {
+  title: 'Are You Sure?',
+  body: 'This link is taking you to the following website',
+  goBack: 'Go Back',
+  visitSite: 'Visit Site'
+}
+
+export const LeavingAudiusModal = () => {
+  const [{ isOpen, link }, , close] = useLeavingAudiusModal()
+  const handleOpen = useCallback(() => {
+    window.open(link, '_blank', 'noreferrer,noopener')
```

Comment:
I hate using `window.open` but I don't think we have a way to make a button link

Reply:
we have a _ton_ of usages of window.open in the web codebase. maybe worth us considering a helper for it all so we can keep the same _blank, noreferrer, etc. behavior